deepgaze/haar_cascade.py

```diff
@@ -146,7 +146,7 @@ def _findFrontalFace(self, inputImg, scaleFactor=1.1, minSizeX=30, minSizeY=30,
             scaleFactor=scaleFactor,
             minNeighbors=minNeighbors,
             minSize=(minSizeX, minSizeY),
-            flags=cv2.cv.CV_HAAR_SCALE_IMAGE
+            flags=cv2.CASCADE_SCALE_IMAGE
```

Comment:
Oh , awesome. Thank you for the fix 👍 

@mpatacchiola will merge it when he logs in.

Reply:
I think this should be handled explicitly at some point in the pipeline. 
I mean, we could check if the version of OpenCV is 2 or 3 and use the correct flag.

However, recall that the master branch of Deepgaze is currently compatible only with OpenCV 2, meaning that this patch cannot be merged on the master. We should instead check if it has been correctly implemented on the branch 2.0 which is compatible with OpenCV 3.

Reply:
Oh , correct. @mpatacchiola We have to still port haar cascades to 2.0 . I think we could add this explicit fix for 2 or 3. The thing is that I have seen certain parts of the codebase work with 3.0 already.

Reply:
This has now been added to branch 2.0